database.hh

```diff
@@ -76,6 +76,9 @@
 #include "query_class_config.hh"
 #include "absl-flat_hash_map.hh"
 #include "utils/cross-shard-barrier.hh"
+#include <boost/icl/interval.hpp>
+#include <boost/icl/interval_map.hpp>
+#include "timestamp.hh"
```

Comment:
Let's use indirection to avoid including this in a header. It's very large.

Reply:
I assume you mean interval*.hpp. timestamp.hh is tiny.

Reply:
Yes, the boost stuff. Not only is timestamp.hh tiny, it's already included everywhere.